software-peer-review-guide/editor-in-chief-guide.md

```diff
@@ -40,101 +40,78 @@ the scope and readiness of a package to be reviewed, they may opt to assign an e
 
 When a new package is submitted for review, the Editor in Chief will:
```

Comment:
IMPORTANT TODO: We should add some documentation here about the EIC workflow and timeline. 

* checks performed all at once. 
* expectations for when the checks come in communicated clearly to the author
* @NickleDave add whatever i missed here from our emails :)

Reply:
Furthermore, based on previous experience, checks should be performed in the following order:
- [ ] Ensure that the package states it can be installed according to an accepted method as stated in our guide (link when this exists). I.e., off of PyPI or if not possible due to dependencies through a community organized conda channel such as conda-forge or bioconda. A package that needs to be "installed" by cloning the source code, for example, or by using a private conda channel would not be acceptable
- [ ] Second, directly verify that the package can be installed and run as it claims. This should not be an exhaustive check such as a reviewer would perform but at a bare minimum should involve confirming that a package manager such as pip or conda is able to resolve the dependencies in a reasonable amount of time, and that any basic functionality like a command-line interface is functional, i.e. by running `package --help` or `package --version`. Where any example snippets or quickstart guides are provided, these should be run too, but again only with the goal of confirming that they run without obvious fatal errors.

Reply:
ok got it. i'm going to work on this a bit more either later today or tomorrow. knowing that more edits could still come in through the end of the week :)

Reply:
ok i think the above is a lot. i added this to the editor checks

```
- [ ] **Installation** The package can be installed from a community repository such as PyPI (preferred), and/or a community channel on conda (e.g. conda-forge, bioconda) 
    - [ ] The package installs and imports in an environment `import package-name`
```

So essentially if the EIC tests that the package indeed can be pip installed or conda installed and it works i think that is enough. 

If the package has tutorials or docstrings it could be tests in CI or setup with sphinx gallery and then we just look at the CI build to see if things run. But i think that is where reviewers should step in rather than be exhaustive early on. 

If you can install it and import package name i think that's plenty. BTW that is totally an automate-able check!

Reply:
installation instructions for installing the package should be in the readme so your first check should get satisfied there.